### PR TITLE
[CARE-1531] Improve code style

### PR DESCRIPTION
**Pull request by oleg-semyonov**
**Key changes:**
- Updated arbitrary Prettier config with `@prezly/code-style` config. Although the configs were pretty much identical, there were many files that had to be fixed by Prettier 🤷🏻 
    - [See the Pretiter auto-fix changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/d1f038151e9cd779cab5cf69b2bda392b3fc1c03)
- Added ESLint with `@prezly/code-styles` config. The codebase is pretty out-of-date in terms of code-style, so I had to suppress some of the rules to not spend too much time on refactoring.
    - [See the auto-fix changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/bc4a360ccb2dc3073142d550f574c49c9f9ced41)
    - [See additional manual changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/9ce4b4f699ff527ae6281ad0860ebc42380901d3)
- Added Stylelint with `@prezly/code-styles` config.
    - [See the auto-fix changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/1db13ac88e88b8937bfc10e7365f9a479dc9e79b)
    - [See additional manual changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/034efe7950d502e15fcd5d8ff4751177d84a69f8)
- Replaced `prettier` GitHub workflow with `lint` workflow, as ESLint does Prettier checks as well.
- Fixed issues with invalid `example/tsconfig.json` rules. [See changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/48be2b14b16f1b90fa4c40edcce761120b2ecd27)
- Use modern React JSX transform, removed all `import React from 'react'`. [See changes](https://github.com/prezly/content-renderer-react-js/pull/52/commits/0e4bef108d06c15eb5b055d84bb179f028e17390)

[See diff without whitespace](https://github.com/prezly/content-renderer-react-js/pull/52/files?diff=split&w=1)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .idea/
```
Comment (e1himself) [inline .gitignore:1]: I suggest adding `package-lock.json` here.

Comment (oleg-semyonov) [reply .gitignore:2]: Updated in 5207351ef035d3f82de502c709cdb899eaf0c478

Comment (oleg-semyonov) [reply .gitignore:2]: Just a note here - everybody will need to delete `node_modules` and `package-lock.json` on their local env and re-install the deps.

File: .loki/reference/chrome_iphone7_Elements_Gallery_Contained_Small_Padding.png
Comment (oleg-semyonov) [inline .loki/reference/chrome_iphone7_Elements_Gallery_Contained_Small_Padding.png:1]: Hey @e1himself, I updated the Loki snapshots multiple times, but the check still fails.
I believe Loki does more harm than good, so I logged [CARE-1693](https://linear.app/prezly/issue/CARE-1693/consider-using-chromatic-on-content-render-react-js-repo) with a proposal to switch to Chromatic.

Would it be okay to ignore this check for now and merge this PR in its current state?
I could address the E2E issues later this week

Comment (e1himself) [reply .loki/reference/chrome_iphone7_Elements_Gallery_Contained_Small_Padding.png:1]: > Would it be okay to ignore this check for now and merge this PR in its current state?
> I could address the E2E issues later this week

Sure :+1:

**Pull request merged**